src/index.d.ts

```diff
@@ -184,7 +184,7 @@ declare namespace lambda {
 
   export interface AlphaClient extends AxiosInstance {
     raw<T = any>(event: {}, environment: Environment, handler: string): Promise<T>;
-    graphql<T = any>(path, query, variables, config): AxiosPromise<T>
```

Comment:
The `any`s here come from `query` and `variables` being passed to alpha.post (axios.post) as `alpha.post(path, { query, variables }, config);`, where that object is defined by `axios` to be an optional `any`.